src/Microsoft.OpenApi/Extensions/EnumExtensions.cs

```diff
@@ -42,5 +43,107 @@ public static string GetDisplayName(this Enum enumValue)
             var attribute = enumValue.GetAttributeOfType<DisplayAttribute>();
             return attribute == null ? enumValue.ToString() : attribute.Name;
         }
+
+        /// <summary>
+        /// Gets the enum display name for <see typeparamref="T"/> without the use of reflection. 
+        /// </summary>
+        /// <typeparam name="T">The type of the enum value.</typeparam>
+        /// <param name="enumValue">The enum value.</param>
+        /// <returns>The display string to use.</returns>
+        public static string GetDisplayName<T>(this T enumValue) where T : Enum
```

Comment:
I made this API public in keeping with the previous one, but I'd prefer to keep it internal. Is there a strong need for package users to be able to invoke this method?